_includes/head.html

```diff
@@ -1,7 +1,7 @@
 <head>
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <title>Coopdevs</title>
+  <title>{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}</title>
```

Comment:
Where is `site.title` defined?

In any case I would recommend doing something like this: 

```suggestion
  <title>{{ site.title }} - {% if page.title %}{{ page.title }}{% endif %}</title>
```

(and defining `site.title` if it's not defined). Careful, this change would render "Coopdevs - Coopdevs" in the home page.